src/licensure.rs

```diff
@@ -57,6 +59,17 @@ impl Licensure {
                 }
             }
 
+            let shebang_re: Regex = Regex::new(r"^#!.*\n").unwrap();
```

Comment:
Can you use `expect` here instead of unwrap. A good message would be `"shebang regex didn't compile!"`

Reply:
Updated to expect.